warehouse/query-core/src/main/java/datawave/query/jexl/functions/ContentFunctionsDescriptor.java

```diff
@@ -424,9 +424,12 @@ public static JexlNode distributeFunctionIntoIndexQuery(JexlNode function, JexlN
             List<JexlNode> components = new LinkedList<>();
             
             JexlNode deref = JexlASTHelper.dereference(indexQuery);
-            if (deref instanceof ASTAndNode) {
-                // single component
+            if (deref instanceof ASTAndNode || deref instanceof ASTEQNode) {
+                
+                // it is possible that an index query contains only a single equality node
+                // i.e., "phrase('foo', 'foo')" would produce a single equality node "FOO == 'bar'"
                 components.add(deref);
+                
             } else if (deref instanceof ASTOrNode) {
                 // multiple components
                 for (JexlNode child : JexlNodes.children(deref)) {
```

Comment:
On line 439 theres an exception whose message should be updated to include EQ nodes.

Reply:
ahh, good catch